CONTRIBUTING.md

```diff
@@ -159,3 +164,244 @@ This repository uses Dependabot to be notified automatically of available update
+
+## A week in the life of a BOM maintainer
+
+A BOM maintainer is typically in charge of BOM maintenance for 2 weeks. 
+
+Typically as a BOM maintainer, you'll be working directly with the `jenkinsci/bom` repository. Said differently, think hard before using your fork of the `bom` repository.
+
+### Task handling
+
+#### Dependabot created PRs
+
+This will probably be the majority of work you'll do.
+
+In a perfect world, a Dependabot PR will just auto-merge into `master` and you won't have to do anything.
+
+In a not so perfect world, a Dependabot PR will fail to build. Most of the time, it's because a plugin is too new for older LTS lines. The way you'll resolve this issue is to pin the older version to the correct LTS line.
+
+[!TIP] 
+If you do have to do work on a PR, make sure to assign the PR to yourself so others can see that you are actively looking at the PR.
+
+The easiest way to work on the PR is to use the `gh` CLI to checkout the PR:
+
+* `gh pr checkout <PR id>`
+
+Then you can work on the PR. Once done, push your changes back to the PR. If everything is successful, the PR will auto-merge. At this point, you can delete the local branch:
+
+* `git branch -D <branchName>`
+
+#### Manually created PRs
+
+When there is a manually generated PR, there's probably a pretty good chance as a BOM maintainer you won't have to do anything. The person opening the PR should open the PR as `draft`. As a BOM maintainer, feel free to look at a `draft` PR, but don't spend much time on it.
+
+However, if the person reaches out for help, be sure to help them.
+
+### Day of week tasks
+
+#### Friday (BOM release day)
+
+* run `bom-lock-master <issueId>` where `<issueId>` is the issue that you created on Thursday
+* verify that job started at [ci.jenkins.io](https://ci.jenkins.io/job/Tools/job/bom/view/change-requests/)
+* run `bom-job-running <issueId> <buildNumber>`
+* wait for build to make it through the `prep` stag then (typically) take a 1.5-2 hr break
+* [LOOP] if there are any failures, fix until everything is successful
+* run `bom-release-issue-add-release-comment <issueId> <releaseNumber>`
+* run `bom-unlock-master <issueId>`
+* edit the auto-generated release notes
+  * remove `<!-- Optional: add a release summary here -->`
+  * remove `<details>`
+  * remove `<summary>XYZ changes</summary>`
+  * remove `</details>`
+* run `bom-release-issue-close issueId`
+
+#### Saturday/Sunday/Monday
+
+* business as usual tasks
+ 
+#### Tuesday (full test day)
+
+* TODO: add the Tuesday tasks
+
+#### Wednesday
+
+* business as usual tasks
+
+#### Thursday (Prep for BOM release)
+
+* run `bom-release-issue-create yyyy-MM-dd`
+* on the newly created issue, set `Type` to `Task`
+* Locally run tests for `warnings-ng` for all current LINEs and weekly (`bom-test-all-lines warnings-ng`)
+
+## Using `gh` CLI 
+
+As someone that is "on-call" for managing BOM, there are a few helpful aliases/scripts that you can create to make your life easier.
```

Comment:
1. I would just put these in the repository
2. I don't see why these aren't just called weekly via GitHub actions